codegen/src/clusters/NumberConstants.ts

```diff
@@ -21,8 +21,28 @@ import {
     UINT64_MAX,
     UINT8_MAX,
 } from "#general";
-import * as Elements from "#model";
-import { Model } from "#model";
+import {
+    attribId,
+    clusterId,
+    commandId,
+    devtypeId,
+    endpointNo,
+    epochS,
+    epochUs,
+    eventId,
+    fabricId,
+    fabricIdx,
+    groupId,
+    Model,
+    nodeId,
+    percent,
+    percent100ths,
+    posixMs,
+    subjectId,
+    systimeMs,
+    systimeUs,
+    vendorId,
+} from "#model";
```

Comment:
Is this a bug in eslint? Or is there a reason we can't import default to access names exports?

Reply:
It seems that eslint can not "see" the names with multiple * exports are imported. So he says that the member is not in the imported namespace. I could revert and disable this one rule?

But because you access each single element alone anyway I thought it is equivalent

Reply:
It is equivalent, a little verbose but it's fine. But I would disable that rule regardless, we don't need eslint to be a second buggy type checker 🙂

Reply:
lets disable when we stumble again over it